p/themes/Origine/origine.css

```diff
@@ -743,14 +743,14 @@ a.btn {
 	background: #0084cc;
 	text-align: center;
 	font-size: 0.9em;
+	padding: 1em;
 }
 
 #new-article:hover {
 	background: #06c;
 }
 
 #new-article > a {
-	line-height: 3em;
```

Comment:
Sidenote but line-height should not normally (if ever) use units.

Reply:
this line I deleted. It is not added ;)

Reply:
I know, but it's important not to repeat this mistake.